client/reducers/scriptsUpdate.js

```diff
@@ -0,0 +1,33 @@
+import { fromJS } from 'immutable';
```

Comment:
This should be improved. The state of each script should be tracked in the scripts reducer. Every script can be a field in this object https://github.com/auth0-extensions/auth0-delegated-administration-extension/blob/dev/client/reducers/scripts.js#L9

Eg:

```
records: {
  filter: {
    loading: false, 
    error: 'foo'
  },
  access: {

  },
  ...
}